lte/gateway/c/core/oai/tasks/amf/include/amf_client_servicer.h

```diff
@@ -194,6 +199,12 @@ class AMFClientServicer : public AMFClientServicerBase {
                              amf_ue_ngap_id_t ue_id) override {
     return true;
   }
+
+  int n11_update_location_req(const s6a_update_location_req_t* const ulr_p) {
+    return RETURNok;
+  }
+
+  bool set_smf_notification(SetSmNotificationContext& notify) { return true; }
```

Comment:
```suggestion
  bool set_smf_notification(const SetSmNotificationContext& notify) { return true; }
```

Reply:
Making this change, required changing the API at `magma/lte/gateway/c/core/oai/lib/n11/SmfServiceClient.h`. Have made the changes. Comment addressed.